docs/install-nuget-client-tools.md

```diff
@@ -38,6 +38,25 @@ The [MSBuild CLI](reference/msbuild-targets.md) also restores and creates packag
 
 Package Manager Console commands work only within Visual Studio on Windows and don't work within other PowerShell environments.
 
+## Support policy
+
+The most recent version of NuGet.exe is fully supported and can be relied on for bug fixes, updates, and enhancements.
+
+Out-of-support NuGet.exe versions will be unlisted from [tools.json](./api/tools-json.md).
+
+For more information on NuGet.exe's support policy, see the [Microsoft Modern Lifecycle Policy](https://aka.ms/lifecycle).
+
+### Patch Releases
+
+Patched versions of NuGet.exe will be released exclusively when critical security fixes are required for a long-term support (LTS) version of Visual Studio or .NET SDK.
+
+All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [https://aka.ms/opensource/security/create-report].
```

Comment:
```suggestion
All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [MSRC's report page](https://aka.ms/opensource/security/create-report).
```

Inspired by https://learn.microsoft.com/en-us/nuget/reference/nuget-client-sdk#support-policy

Reply:
I accidentally copied over it. Restored now.